projects/Mallard/src/helpers/fetch.ts

```diff
@@ -22,7 +22,9 @@ const fetchFromApiSlow = async <T>(
     } = {},
 ): Promise<T> => {
     const apiUrl = await getSetting('apiUrl')
-    return fetch(apiUrl + path)
+    const url = apiUrl + path
+    __DEV__ && console.log(`fetching ${url}`)
```

Comment:
I'm using `react-native-debugger` where I can see the network requests, headers, headers and all ... maybe that might solve this issue?

Reply:
it ABSOLUTELY will lets get rid of this